teeio-validator/include/ide_test.h

```diff
@@ -294,19 +248,13 @@ typedef struct {
   int cfg_space_fd;
   uint32_t ecap_offset;
   uint32_t doe_offset;
-  PCIE_IDE_CAP ide_cap;
 
   // kcbar related data
   uint8_t *mapped_kcbar_addr;
   int kcbar_fd;
-  INTEL_KEYP_PCIE_STREAM_CAP stream_cap;
 
-  // ide_id
-  uint8_t ide_id;
-
-  // rid/addr assoc_reg_block
-  PCIE_SEL_IDE_RID_ASSOC_REG_BLOCK rid_assoc_reg_block;
-  PCIE_SEL_IDE_ADDR_ASSOC_REG_BLOCK addr_assoc_reg_block;
+  // private data, for example for PCIE IDE or CXL IDE.    
+  void *priv_data;
 } ide_common_test_port_context_t;
```

Comment:
why we need change this structure?

Why not keep it unchanged and just define a new `cxl_ide_common_test_port_context_t` for cxl_ide?